mods/d2k/rules/structures.yaml

```diff
@@ -648,6 +660,15 @@ starport:
 	ProvidesPrerequisite@harkonnen:
 		Prerequisite: starport.harkonnen
 		Factions: harkonnen
+	ProvidesPrerequisite@atreides_combat:
+		Prerequisite: starport.atreides_combat
```

Comment:
What is the purpose of `starport.atreides_combat`? If it is for the combat tanks in the `starport.yaml`, they are already by `starport.atreides` ([Link](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/starport.yaml#L64)).

Reply:
https://github.com/OpenRA/OpenRA/pull/14459/files#diff-ba4c66055d3a27a599f4657d0b682cc2R64

Reply:
Ahh sorry, I searched for it for sure but somehow screwed up.